tests/test_scala.py

```diff
@@ -0,0 +1,29 @@
+def test_java_exists(Command):
+	java = Command("java -version")
+	assert java.rc == 0
+
+
+def test_scala_exists(Command, Ansible, SystemInfo):
+	scala_version = Command(
+		"scala -e \"println(util.Properties.versionNumberString)\"")
+	assert scala_version.rc == 0
+
+def test_scala_version(Command, Sudo):
+	with Sudo(user='root'):
+		# Extract target scala version from ansible facts
+		target_scala_version = Command.check_output(
```

Comment:
Did you try doing anything with the `Ansible()` TI module?

I setup a fixture like this:

```python
import os.path
import pytest


@pytest.fixture()
def AnsibleDefaults(Ansible):
    return Ansible("include_vars", os.path.join("./defaults/main.yml"))["ansible_facts"]
```

Which provides a data structure like:

```python
{u'scala_version': u'2.11.1', u'scala_sbt_version': u'0.13.6'}
```

I think this might allow us to test the configured variables from within TI without a template.